component/libfuse/libfuse_handler_test_wrapper.go

```diff
@@ -77,7 +77,7 @@ func newTestLibfuse(next internal.Component, configuration string) *Libfuse {
 	config.ReadConfigFromReader(strings.NewReader(configuration))
 	libfuse := NewLibfuseComponent()
 	libfuse.SetNextComponent(next)
-	libfuse.Configure()
+	libfuse.Configure(true)
```

Comment:
Similar changes are needed for fuse2 as well, it's not building right now.

Reply:
good catch